.devcontainer/Dockerfile

```diff
@@ -1,3 +1,7 @@
 # Make sure RUBY_VERSION matches the Ruby version in .ruby-version
 ARG RUBY_VERSION=3.3.4
 FROM ghcr.io/rails/devcontainer/images/ruby:$RUBY_VERSION
+
+# Ensure binding is always 0.0.0.0
+# Binds the server to all IP addresses of the container, so it can be accessed from outside the container.
+ENV BINDING="0.0.0.0"
```

Comment:
Could be part of https://github.com/rails/devcontainer/blob/f4c005990ba0e48b68da736d605a0f94c745a466/images/ruby/.devcontainer/Dockerfile or a feature